flask_monitor/metrics.py

```diff
@@ -42,19 +31,38 @@ def register_metrics(app, buckets=None, error_fn=None):
     Before CPython 3.6 dictionaries didn't guarantee keys order, so callbacks
     could be executed in arbitrary order.
     """
+
+    if app.config.get("METRICS_ENABLED", False):
+        return app
+    app.config["METRICS_ENABLED"] = True
+    app.logger.info('Metrics enabled')
```

Comment:
@Ziul, just to confirm, this is an approach to make the activation of the metrics registration middleware unique, right?

Reply:
Yes! This avoids running lines 99 and 100, where the middleware is registered.

Reply:
👍